src/dotnet-publish-iis/Program.cs

```diff
@@ -32,7 +33,13 @@ public static int Main(string[] args)
                     return 2;
                 }
 
-               return new PublishIISCommand(publishFolder, projectPath.Value, webRootOption.Value()).Run();
+                Reporter.Output.WriteLine($"Configuring project published to '{publishFolder}' for use with IIS");
```

Comment:
publishFolder can get long and obscure your message. How about 
`"Configuring the following project for use with IIS: '{publishFolder}'"`